domains/certificates/chart/values.yaml

```diff
@@ -7,3 +7,14 @@ postgres:
     name: sibedge/postgres-plv8
     tag: 15.0-3.1.4-alpine
   size: 10Gi
+rabbitmq-cluster-operator:
+  extraDeploy:
+    - apiVersion: rabbitmq.com/v1beta1
+      kind: RabbitmqCluster
+      metadata:
+        name: certificate-rabbitmq-cluster
+      spec:
+        replicas: 1
+        rabbitmq:
+          additionalConfig: |
+            log.console.level = debug
```

Comment:
Not sure that we want to have this log.console.level. I suggest that it is removed and we just use the defaults

Reply:
Marked as resolved, as this pr is still a draft and not done yet.